packages/core/docs/guide/authentication.md

```diff
@@ -128,7 +128,64 @@ export default {
   }
 }
 </script>
-```  
+```
+
+## Forgot Password
+
+Usually the process of resetting user password after forgetting consists of two steps:
+
+1. Generate reset password token with an email address.
+
+```vue
+<template>
+  <form @submit.prevent="reset({ email: form.value.email })">
```

Comment:
`form` object doesn't exist in `setup` method

Reply:
Corrected